x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/accounts_stats_collector.ts

```diff
@@ -0,0 +1,243 @@
+            },
+            size: 1,
+            sort: {
+              '@timestamp': 'desc',
+            },
+          },
+        },
+        passed_findings_count: {
+          filter: {
+            bool: {
+              filter: [
+                {
+                  bool: {
+                    should: [
+                      {
+                        term: {
+                          'result.evaluation': 'passed',
+                        },
+                      },
+                    ],
+                    minimum_should_match: 1,
+                  },
+                },
+              ],
+            },
+          },
+        },
+        failed_findings_count: {
+          filter: {
+            bool: {
+              filter: [
+                {
+                  bool: {
+                    should: [
+                      {
+                        term: {
+                          'result.evaluation': 'failed',
+                        },
+                      },
+                    ],
+                    minimum_should_match: 1,
+                  },
+                },
+              ],
+            },
+          },
+        },
+        resources: {
+          filter: {
+            bool: {
+              filter: [
+                {
+                  bool: {
+                    should: [
+                      {
+                        term: {
+                          'resource.sub_type': 'Pod',
+                        },
+                      },
+                    ],
+                    minimum_should_match: 1,
+                  },
+                },
+              ],
+            },
+          },
+          aggs: {
+            pods_count: {
+              cardinality: {
+                field: 'resource.id',
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+
+  size: 0,
+  _source: false,
+});
+
+const getCspmAccountsStats = (
+  aggregatedResourcesStats: AccountsStats,
+  logger: Logger
+): CspmAccountsStats[] => {
+  const accounts = aggregatedResourcesStats.accounts.buckets;
+
+  const cspmAccountsStats = accounts.map((account) => {
+    return {
+      account_id: account.key,
+      latest_findings_doc_count: account.doc_count,
+      posture_score: calculatePostureScore(
+        account.passed_findings_count.doc_count,
+        account.failed_findings_count.doc_count
+      ),
+      passed_findings_count: account.passed_findings_count.doc_count,
+      failed_findings_count: account.failed_findings_count.doc_count,
+      benchmark_name: account.benchmark_name.top[0].metrics['rule.benchmark.name'],
+      benchmark_id: account.benchmark_id.top[0].metrics['rule.benchmark.id'],
+      benchmark_version: account.benchmark_version.top[0].metrics['rule.benchmark.version'],
+      agents_count: account.agents_count.value,
+      nodes_count: account.nodes_count.value,
+      pods_count: account.resources.pods_count.value,
+    };
+  });
```

Comment:
You remove the return here if you want

```suggestion
  const cspmAccountsStats = accounts.map((account) =>  ({
      account_id: account.key,
      latest_findings_doc_count: account.doc_count,
      posture_score: calculatePostureScore(
        account.passed_findings_count.doc_count,
        account.failed_findings_count.doc_count
      ),
      passed_findings_count: account.passed_findings_count.doc_count,
      failed_findings_count: account.failed_findings_count.doc_count,
      benchmark_name: account.benchmark_name.top[0].metrics['rule.benchmark.name'],
      benchmark_id: account.benchmark_id.top[0].metrics['rule.benchmark.id'],
      benchmark_version: account.benchmark_version.top[0].metrics['rule.benchmark.version'],
      agents_count: account.agents_count.value,
      nodes_count: account.nodes_count.value,
      pods_count: account.resources.pods_count.value,
    }));
```